app/workers/commit_monitor_handlers/commit_range/rubocop_checker/message_builder.rb

```diff
@@ -14,6 +14,20 @@ def comments
     message_builder.comments
   end
 
+  # requirements to "create_status(repo, sha, state, options)"
+  def commit_status
```

Comment:
Why is this method under the `MessageBuilder`? Maybe this can be moved to a better place.

Reply:
Agreed.  This feels like it should live somewhere else.

Reply:
@skateman @Fryguy The `commit_status` method is now in the `MessageBuilder`.